File: seungyeop/22_12_22_02.java
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        if (a > b) {
            int c = a;
            a = b;
            b = c;
        }
        for (; a <= b; a++) {
            answer += a;
        }
        return answer;
    }
}

File: minseon/22_12_22_02.py
# 없는 숫자 더하기


def solution(numbers):
    answer = 0
    num = [0,1, 2, 3, 4, 5, 6, 7, 8, 9]
    for x in num:
        if x not in numbers:
            answer += num[x]
    return answer
